docs/developer/stock-allocation.mdx

```diff
@@ -148,7 +148,11 @@ the [`updateWarehouse`](api-reference/products/mutations/update-warehouse.mdx) m
 There is an option to change the following warehouse fields: `name`, `slug`, `email`, `address` and
 also define privacy and `click and collect` options.
 
-We can define warehouse as public or private. Public warehouse allows you to query the stock information publicly in the API. Private warehouse requires permission.
+#### Public vs. private warehouses
+
+We can define warehouse as public or private. Public warehouse allows you to query the stock information publicly in the API. Private warehouse requires permission. [`quantityAvailable`](api-reference/products/objects/product-variant.mdx#productvariantquantityavailableint--) is the sum of the available quantity in all public and private warehouses.
```

Comment:
Maybe? As it's not clear what type `quantityAvailable` belongs to, also it does not look good when the sentence starts with a small letter.
```suggestion
We can define warehouse as public or private. Public warehouse allows you to query the stock information publicly in the API. Private warehouse requires permission. The [`quantityAvailable`](api-reference/products/objects/product-variant.mdx#productvariantquantityavailableint--) field on `ProductVariant` type is the sum of the available quantity in all public and private warehouses.
```